### ISSUE TITLE
Cascaded Select not working with Material

### ISSUE THREAD

**Issue by subratawebops** [formly-js/ngx-formly#611]
I'm using cascaded select using the material design its not working but when I switch into bootstrap its working. I'm using the same example that you have in your example. Are you supporting material???

```
import { Component, OnInit } from '@angular/core';

import { HttpService } from './http.service';
import {FormGroup} from '@angular/forms';
import {FormlyFieldConfig} from '@ngx-formly/core';
import { FormlyFormOptions } from '@ngx-formly/core';
@Component({
  selector: 'app-root',
  templateUrl: './app.component.html',
  styleUrls: ['./app.component.css']
})
export class AppComponent implements OnInit {
    form = new FormGroup({});
    formModel: any  = {};
    options: FormlyFormOptions = {};
    formFields: FormlyFieldConfig[] = [
        {
            fieldGroupClassName: 'flex-container',
            fieldGroup: [
                {
                    className: 'mat-form-field',
                    type: 'input',
                    key: 'firstName',
                    templateOptions: {
                      label: 'First Name',
                    }
                },
                {
                    className: 'mat-form-field',
                    type: 'input',
                    key: 'lastName',
                    templateOptions: {
                      label: 'Last Name',
                    },
                    expressionProperties: {
                        'templateOptions.disabled': '!model.firstName',
                    }
                },
                {
                    className: 'col-xs-12',
                    type: 'select',
                    key: 'select1',
                    defaultValue: 'a',
                    templateOptions: {
                      label: 'select box 1',
                      options: [
                        { value: 'a', label: 'A' },
                        { value: 'b', label: 'B' },
                      ]
                    }
                  },
                  {
                    className: 'col-xs-12',
                    type: 'select',
                    key: 'select2',
                    lifecycle: {
                      onInit: (form, field) => {
                        const updateOptions = (v) => {
                          if (v.select1 === 'a') {
                            this.formFields[1].templateOptions.options = [
                              { value: 'a1', label: 'A > 1' },
                              { value: 'a2', label: 'A > 2' },
                            ]
                          }
                          if (v.select1 === 'b') {
                            this.formFields[1].templateOptions.options = [
                              { value: 'b1', label: 'B > 1' },
                              { value: 'b2', label: 'B > 2' },
                            ]
                          }
                        };
                        updateOptions(form.value);
                        this.form.valueChanges.subscribe(v => updateOptions(v));
                      },
                    },
                    templateOptions: {
                      label: 'select box 2',
                      options: [
                        { value: '' },
                      ]
                    },
                  }
            ]
        }
  ];
    submit() {
        console.log(this.formModel);
    }
  constructor(
      private _HttpService: HttpService
  ) { }
  ngOnInit() {
    this._HttpService.fetchForm().subscribe(response => {
        console.log(response);
    });
  }
}

```

**Comment by subratawebops** [comment]
@aitboudad Can you tell me where I'm going wrong?

**Comment by aitboudad** [comment]
@subratawebops it's a bug, I'm on it!

**Comment by subratawebops** [comment]
@aitboudad Thanks a ton.

**Comment by aitboudad** [comment]
fixed in `2.0.0-rc.4`
